build_projects/shared-build-targets-utils/Utils/HostVersion.cs

```diff
@@ -24,7 +24,8 @@ public class HostVersion : Version
         public string LatestHostPolicyVersionNoSuffix => $"{Major}.{Minor}.{Patch}";
         public string LatestHostPrerelease => ReleaseSuffix;
         public string LatestHostBuildMajor => $"{CommitCountString}";
-        public string LatestHostSuffix => $"{ReleaseSuffix}-{LatestHostBuildMajor}-00";
+        public string LatestHostBuildMinor => "00";
```

Comment:
my impression was that it was in case we needed to manually patch a package, but I agree the commit count should be sufficient.